frontend/src/utils/helpers.js

```diff
@@ -49,6 +49,13 @@ export const formatDate = (dateString) =>
   dateString !== "None" &&
   moment(dateString, "YYYY-MM-DD").format("MMM DD YYYY");
 
+// Function to format an API date string to human readable
+export const formatFullDateTime = (dateString) =>
+  dateString &&
+  dateString !== "9999-12-31" &&
+  dateString !== "None" &&
+  moment(dateString).format("MMM DD YYYY");
```

Comment:
How does this dateString differ than the one above?

Reply:
It is in a UTC full format so YYYY-MM-DDTHH:mm:SSZ

With this change all dates will be returned in this way so i didn't want to modify the currently working version and just added this to bridge the gap for now.

Reply:
Thanks for the update!